README.md

```diff
@@ -116,7 +116,19 @@ file][]) and you're good to go:
   https://jestjs.io/docs/en/configuration.html#setupfilesafterenv-array
 
 ```javascript
+// In your own jest-setup.ts (or any other name) 
 import '@testing-library/jest-dom'
+
+// jest.config.js add:
+setupFilesAfterEnv: ["<rootDir>/jest-setup.ts"]
+
+// Include your jest-setup.ts if you havent already
+// tsconfig.json
+  "include": [
+    ...
+    "./jest-setup.ts"
+  ],
+
 ```
 
 > Note: If you're using TypeScript, make sure your setup file is a `.ts` and not
```

Comment:
The changes in this PR so far assume usage of TypeScript. Maybe leave it all js-centric, and add a sub-header down at this point where we summarize the parts related to TS (different extension, and the snippet to add to tsconfig.json, etc.)

Reply:
You're totally right. I've had my head full of TS and forgot about JS lol.